lib/elixir/lib/option_parser.ex

```diff
@@ -112,6 +113,15 @@ defmodule OptionParser do
       iex> OptionParser.parse(["--no-op", "path/to/file"], switches: [op: :boolean])
       {[op: false], ["path/to/file"], []}
 
+  ### Parsing undefined switches
+
+  By default, only arguments that have defined atom representation will be parsed.
+  This happens because creating atoms at run-time is considered to be unsafe,
+  but you still can force parse them by passing `allow_nonexistent_atoms: true`
```

Comment:
Perhaps "you still can force creation of atoms by passing" should be expressive and clear, WDYT?